### DISCUSSION TITLE
Duda respecto a REPORTE-PEDIDOS y REPORTE-NAVE

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#13] [Tarea 0] [answered]
Tengo una duda respecto al resultado esperado de REPORTE-PEDIDOS, ya que en el enunciado se menciona que "Si
 una nave no tiene pedidos pendientes, esa nave no se debe imprimir.", pero en el ejemplo 1-3 sucede que la nave 2 no tiene pedidos pero aún así se imprime NAVE 2, entonces no estoy seguro cual de las dos instrucciones debiéramos seguir...

Con REPORTE-NAVE sucede algo similar, el enunciado dice que hay que incluir el ID de la nave en el output, pero en los ejemplos no están incluido

**Comment by PabloNavarroY** [comment, marked as answer]
Ambas cosas fueron errores en la generación de los outputs esperados. Te aviso cuando actualizemos los tests.

**Reply by tomastrivino** [reply]
Gracias por el aviso!

**Reply by MaxAl100** [reply]
Ya! También agradecería si añaden una descripción de qué sucede con REPORTE-PEDIDOS cuando ninguna nave tiene algún pedido.
